cmd/pipeline/destroy.go

```diff
@@ -91,26 +94,27 @@ func ExecuteDestroyPipeline(ctx context.Context, opts *DestroyOptions) error {
 		opts.Name = getPipelineName(repo)
 	}
 
-	resp, err := destroyPipeline(ctx, opts.Name, opts.DestroyVolumes)
-	if err != nil {
+	resp, err := pc.destroyPipeline(ctx, opts.Name, opts.DestroyVolumes)
+	if err != nil && !oktetoErrors.IsNotFound(err) {
 		return err
 	}
 
 	if !opts.Wait {
 		oktetoLog.Success("Repository '%s' scheduled for destruction", opts.Name)
 		return nil
 	}
-
-	if err := waitUntilDestroyed(ctx, opts.Name, resp.Action, opts.Timeout); err != nil {
-		return err
+	if err == nil || !oktetoErrors.IsNotFound(err) {
```

Comment:
I think you can omit the second part of the if. You can include a comment to indicate that if the error was a not found we don't have to wait. I think that `!oktetoErrors.IsNotFound(err)` cannot be true because the line 98 would be true.